### PR TITLE
Added config for kafka-bridge

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

File: ansible/userdata/default_instance_user_data.yaml [hunk 1]
```diff
@@ -91,6 +91,8 @@ coreos:
           ExecStart=/bin/sh -c "etcdctl set /ft/config/binary-writer/bucket '{{binary_writer_bucket}}' > /dev/null;"
           ExecStart=/bin/sh -c "etcdctl set /ft/config/synthetic-image-publication-monitor-aws/test-uuid '{{aws_image_monitor_test_uuid}}' > /dev/null;"
           ExecStart=/bin/sh -c "etcdctl set /ft/config/synthetic-image-publication-monitor-ucs/test-uuid '{{ucs_image_monitor_test_uuid}}' > /dev/null;"
+          ExecStart=/bin/sh -c "etcdctl set /ft/config/kafka-bridge/ucs_queue_proxy_addrs '{{bridging_message_queue_proxy}}' > /dev/null;"
+          ExecStart=/bin/sh -c "etcdctl set /ft/config/kafka-bridge/authorization_key '{{bridging_message_queue_proxy_authorization_key}}' > /dev/null;"
```
Comment (Zapadlo) [inline ansible/userdata/default_instance_user_data.yaml]: Should this be in `_credentials`?

Comment (catalin-ilea) [inline ansible/userdata/default_instance_user_data.yaml]: Done

**Comment by Zapadlo** [comment]
+1

**Comment by martingartonft** [comment]
+1

**Pull request merged**